screens/AccountCloseConfirmForm.js

```diff
@@ -0,0 +1,452 @@
+              				name="name"
+              				component={InputText}
+              				//validate={isRequired}
+              				//placeholder={t('productName')}
+              				placeholder={"??"}
+              				secureTextEntry={false}
+              				height={35}
+            				/>
+                	</View>
+              	</View>
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount actually owned by cash register
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
+              				name="name1"
+              				component={InputText}
+              				//validate={isRequired}
+              				//placeholder={t('productName')}
+              				placeholder={"??"}
+              				secureTextEntry={false}
+              				height={35}
+            				/>
+                	</View>
+              	</View>
+            	</View>         		
+          	</View>
+        	{/* #Credit Card */}
+
+					{/* Invoice */}
+          	<View style={styles.sectionBar}>             
+            	<View style={[styles.tableCellView, {flex: 4}]}>
+              	<TouchableOpacity>
+                	<Text style={styles.sectionBarTextSmall}>
+                  	{t('invoice')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Total number of invoices
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;?</Text>
+                	</View>
+              	</View>                
+            	</View>             		
+          	</View>
+        	{/* #Invoice */}
+
+					{/* Others */}
+          	<View style={styles.sectionBar}>             
+            	<View style={[styles.tableCellView, {flex: 4}]}>
+              	<TouchableOpacity>
+                	<Text style={styles.sectionBarTextSmall}>
+                  	{t('others')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>              	              
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Total number of orders deleted
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
```

Comment:
This shouldn't be a field. It should display the total deleted order using the following field:
close.closingShiftReport.orderCountByState['DELETED'].orderCount

The data structure looks like this for your reference:

 "orderCountByState": Object {
        "COMPLETED": Object {
          "id": "COMPLETED",
          "orderCount": 2,
          "orderState": "COMPLETED",
        },
      }